bertopic/plotting/_topics.py

```diff
@@ -56,6 +57,9 @@ def visualize_topics(
     freq_df = freq_df.loc[freq_df.Topic != -1, :]
     if topics is not None:
         topics = list(topics)
+    elif len(topics) < 5:
+        warnings.warn("Too few topics to display.")
+        return
```

Comment:
Couple of questions:

* What makes you choose the value of < 5 here? Is it working for 4 topics but it is for 5?
* Should we raise an error rather than return nothing after a warning? By returning None, we might also have to change the typing. 
* Could you extend the description a bit more? Perhaps explain that we need more topics in order to properly visualize them?